pkg/assembler/backends/neo4j/src.go

```diff
@@ -197,6 +197,26 @@ func (e *srcNamespaceToName) IdentifiablePropertyNames() []string {
 }
 
 func (c *neo4jClient) Sources(ctx context.Context, sourceSpec *model.SourceSpec) ([]*model.Source, error) {
+
+	// fields: [type namespaces namespaces.namespace namespaces.names namespaces.names.name namespaces.names.tag namespaces.names.commit]
+	fields := getPreloads(ctx)
+	nameRequired := false
+	namespaceRequired := false
+	for _, f := range fields {
+		if f == "namespaces" {
+			namespaceRequired = true
+		}
+		if f == "namespaces.names" {
```

Comment:
nit: may be cleaner to make this a const that gets built, to show the structure, e.g. const ns = "...", names = ns + "." + "names", etc.